setup/install-openroad-rhel7.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+scr_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
+
```

Comment:
```suggestion
```

This looks like a handy shell construction, but we would like to keep the setup shell scripts easy to read if it's not required.